### ISSUE TITLE
Issue on OSX

### ISSUE THREAD

**Issue by MajickTek** [prideout/par#22]
Everything builds and runs fine, but at certain parts has a segfault of 11.
Here are the test infos:

bubbles
~~~~

  par_bubbles_pack
Segmentation fault: 11
~~~~

shapes
~~~~

  cylinders and spheres
    ✓ should fail when the number of stacks or slices is invalid
    ✓ should generate correct number of vertices
    ✓ should have expected number of triangles


  par_shapes_create_plane
    ✓ should not have NaN's


  par_shapes_export
Segmentation fault: 11
~~~~

sprune
~~~~

  par_sprune_overlap
box0 collides with box1
There are now 3 collisions.
Segmentation fault: 11
~~~~

bluenoise works fine, test_linker outputs nothing. test_cpp does nothing.

the file cache tests both work fine and produce the same output.

I tried squares twice, and one time it downloaded the assets and then had a segmentation fault of 11.

the other time I tried it, I got this:
~~~~
Downloading http://github.prideout.net/assets/msquares_island.1024.bin...
test_msquares(24328,0x7fffb8cd6340) malloc: *** error for object 0x1030cf000: pointer being freed was not allocated
*** set a breakpoint in malloc_error_break to debug
Abort trap: 6
~~~~
When I ran it again (thinking it was just a random memory error that would fix with a relaunch), I got the same error code but with a different "object": 
~~~~
test_msquares(24339,0x7fffb8cd6340) malloc: *** error for object 0x10f6d8000: pointer being freed was not allocated
*** set a breakpoint in malloc_error_break to debug
Abort trap: 6
~~~~

this "object" is different every time I relaunch the squares test.

**Comment by prideout** [comment]
Ah, I think all these failures are due to an incorrect current working directory.  I fixed the instructions in commit a0ecff5.

**Comment by MajickTek** [comment]
Thanks a bunch! All tests now work perfectly fine and produce the desired output, no segfaults and no download errors.
